apps/meteor/app/livechat/server/lib/contacts/getAllowedCustomFields.ts

```diff
@@ -0,0 +1,12 @@
+import type { ILivechatCustomField } from '@rocket.chat/core-typings';
+import { LivechatCustomField } from '@rocket.chat/models';
+
+export async function getAllowedCustomFields(): Promise<Pick<ILivechatCustomField, '_id' | 'label' | 'regexp' | 'required'>[]> {
+	return LivechatCustomField.findByScope(
+		'visitor',
+		{
+			projection: { _id: 1, label: 1, regexp: 1, required: 1 },
+		},
+		false,
+	).toArray();
+}
```

Comment:
if this filter is so fancy that deserves a function, it should be added to the model right?

Reply:
This one is part of the code that was split from the large Contacts file into separate files for each function.